menu.txt

```diff
@@ -1,3 +1,3 @@
 dosa
-karam
-oil
\ No newline at end of file
+extra karam
```

Comment:
Do we really need extra karam

Reply:
yes we need. Because ghee neutralize some karam